components/page.jsx

```diff
@@ -125,7 +135,13 @@ var Page = React.createClass({
           <div className="row">
             <Sidebar/>
             <main className="content col-md-9" role="main" id="content" tabIndex="-1">
-              <RouteHandler showModal={this.showModal} hideModal={this.hideModal}/>
+            {
+              React.cloneElement(this.props.children, {
```

Comment:
could you explain why we are doing `cloneElement` vs just passing extra `props` to the existing element?